Scenario sample framework

AlexJerabek opened the pull request:
This creates a space for larger, scenario-driven samples to be shared. It includes one such sample that analyzes web traffic records to determine the user's origin.

o365devx commented:
Docs Build status updates of commit _[b06f444](https://github.com/OfficeDev/office-scripts-docs/commits/b06f444f40c746101147ad4e2b25635b8e163721)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[_dependentPackages/CommonPlugins/tools/JoinTOC.ps1](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/_dependentPackages/CommonPlugins/tools/JoinTOC.ps1) | :x:Error | | [Details](#user-content-402d9d56-ddc10525)
[.openpublishing.publish.config.json](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/.openpublishing.publish.config.json) | :x:Error | | [Details](#user-content-d902ab53-ddc10525)
[]() | :x:Error | | [Details](#user-content-ddc10525)

<a id="402d9d56-ddc10525"></a>
### [_dependentPackages/CommonPlugins/tools/JoinTOC.ps1](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/_dependentPackages/CommonPlugins/tools/JoinTOC.ps1)
 - **Line 82**: **[Error]** `````Error happened when executing plug-in script _dependentPackages/CommonPlugins/tools/JoinTOC.ps1 at step docset_prebuild. Please check your .openpublishing.publish.config.json?on the configuration of the plug-ins or ask for help?in Docs Support teams channel. Exit code: -1073741502`````
 - **Line 217**: **[Error]** `````Error happened when executing plug-in script _dependentPackages/CommonPlugins/tools/JoinTOC.ps1 at step docset_prebuild. Please check your .openpublishing.publish.config.json?on the configuration of the plug-ins or ask for help?in Docs Support teams channel. Please also include the build report and the following error code for our team to troubleshoot: 1`````

<a id="d902ab53-ddc10525"></a>
### [.openpublishing.publish.config.json](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/.openpublishing.publish.config.json)
 - **Line 80**: **[Error]** `````Failed to execute customized script at step docset_prebuild in docset office-scripts-docs. Please check your .openpublishing.publish.config.json?on the configuration of the plug-ins for docset office-scripts-docs step docset_prebuild or ask for help?in Docs Support teams channel. Please also include the build report and the following error code for our team to troubleshoot: 1`````

<a id="ddc10525"></a>
###
 - **Line 138**: **[Error]** `````W:\xbmu-s\.optemp\packages\opbuild.scripts.1.84.4\tools\opbuild\docset.prebuild.ps1 failed to execute`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C13%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202002132231115595-88%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=blnFkwLYqqAU9HyzQGfdzzc3NTjFXphmiZwcaepfs3Y%3D&st=2020-02-13T22%3A27%3A01Z&se=2020-03-15T22%3A32%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

o365devx commented:
Docs Build status updates of commit _[03bf7dc](https://github.com/OfficeDev/office-scripts-docs/commits/03bf7dc367658480402a71b08b768f05fc2b5424)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/scenario-analyze-web-downloads-after.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-after.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-before.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-before.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-table.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-table.png) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.xlsx](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.xlsx) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/sample-scenario-overview.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/sample-scenario-overview.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C14%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202002141712399458-88%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=2snVW0AYN97%2BpdjDRXEpYIVOffd8G7oeOpvkbsdYR1A%3D&st=2020-02-14T17%3A08%3A50Z&se=2020-03-16T17%3A13%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

o365devx commented:
Docs Build status updates of commit _[c8342a8](https://github.com/OfficeDev/office-scripts-docs/commits/c8342a8de0012082daf47657a5cb368662744815)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/scenario-analyze-web-downloads-after.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-after.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-before.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-before.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-table.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-table.png) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.xlsx](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.xlsx) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/sample-scenario-overview.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/sample-scenario-overview.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C14%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202002142147104721-88%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=GddeEsYJaNb%2FKJt6YetmqSJ2lo3pqCr0xIVNd2STysc%3D&st=2020-02-14T21%3A43%3A18Z&se=2020-03-16T21%3A48%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

o365devx commented:
Docs Build status updates of commit _[6a8c789](https://github.com/OfficeDev/office-scripts-docs/commits/6a8c78956d39c2b92505d0b96c263d6eb7952330)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/scenario-analyze-web-downloads-after.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-after.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-before.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-before.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-table.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-table.png) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.xlsx](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.xlsx) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/sample-scenario-overview.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/sample-scenario-overview.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C18%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202002182030148900-88%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=a8keUPQpSg0hsA8%2FEuiuhVYWtWNaO6DA6KsCIeOB8Rw%3D&st=2020-02-18T20%3A26%3A24Z&se=2020-03-20T20%3A31%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

o365devx commented:
Docs Build status updates of commit _[f09c353](https://github.com/OfficeDev/office-scripts-docs/commits/f09c353e77d113e8c6c281b0598546a37b8729c5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/scenario-analyze-web-downloads-after.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-after.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-before.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-before.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-table.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-table.png) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.xlsx](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.xlsx) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/sample-scenario-overview.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/sample-scenario-overview.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C18%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202002182034558962-88%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=B2GhWS96FEI9yEaohnE9EelmyWIvRJ6wbfOZxW%2B8RGE%3D&st=2020-02-18T20%3A31%3A02Z&se=2020-03-20T20%3A36%3A02Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

o365devx commented:
Docs Build status updates of commit _[c950bfc](https://github.com/OfficeDev/office-scripts-docs/commits/c950bfc7465605710aef31a3c185c0a05711d289)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/scenario-analyze-web-downloads-after.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-after.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-before.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-before.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-table.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-table.png) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.xlsx](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.xlsx) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/sample-scenario-overview.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/sample-scenario-overview.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C18%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202002182038398453-88%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=isjQ%2B%2BUR0NnX2nzmzMCGJvJfzvQjVD7Dk0nZw8QFSUM%3D&st=2020-02-18T20%3A34%3A53Z&se=2020-03-20T20%3A39%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

o365devx commented:
Docs Build status updates of commit _[a239bb1](https://github.com/OfficeDev/office-scripts-docs/commits/a239bb1818ea9c91b5007066dd4becea5da5626b)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/resources/scenarios/analyze-web-downloads.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.md) | :warning:Warning | | [Details](#user-content-ef89cd8d-022918bc)
[docs/images/scenario-analyze-web-downloads-after.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-after.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-before.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-before.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-table.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-table.png) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.xlsx](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.xlsx) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/sample-scenario-overview.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/sample-scenario-overview.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/toc.yml) | :white_check_mark:Succeeded | |

<a id="ef89cd8d-022918bc"></a>
### [docs/resources/scenarios/analyze-web-downloads.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.md)
 - **Line 31**: **[Warning]** `````Invalid file link:(~/resources/scenarios/analyze-web-downloads.xlsx).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C18%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202002182043367195-88%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=2pY3oVaoNbxWEDXwpS%2F54urx%2BdiwqZgOMKmZ%2FwuuH5E%3D&st=2020-02-18T20%3A39%3A46Z&se=2020-03-20T20%3A44%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

o365devx commented:
Docs Build status updates of commit _[2fc52fb](https://github.com/OfficeDev/office-scripts-docs/commits/2fc52fb2687081e67a46a3a0935645fb045a954b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docfx.json](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/docfx.json) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-after.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-after.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-before.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-before.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-table.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-table.png) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.xlsx](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.xlsx) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/sample-scenario-overview.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/sample-scenario-overview.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C18%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202002182050229468-88%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=bDdFd%2FghyzTaPDOQBMLaiUNuWbb2RZUYCVGyRnsLjPw%3D&st=2020-02-18T20%3A46%3A31Z&se=2020-03-20T20%3A51%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

o365devx commented:
Docs Build status updates of commit _[c91e0b6](https://github.com/OfficeDev/office-scripts-docs/commits/c91e0b66a128b2b5f16f721d483320aa2d03fec6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docfx.json](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/docfx.json) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-after.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-after.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-before.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-before.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-table.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-table.png) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.xlsx](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.xlsx) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/sample-scenario-overview.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/sample-scenario-overview.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C18%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202002182056182739-88%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=0ktMdMZSbUd5WF2ITG2hmQe7S4acMHGEEUxiYQmHTEQ%3D&st=2020-02-18T20%3A52%3A27Z&se=2020-03-20T20%3A57%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

o365devx commented:
Docs Build status updates of commit _[9e1f061](https://github.com/OfficeDev/office-scripts-docs/commits/9e1f06148617e155b70370156d5bb4554203bc87)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docfx.json](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/docfx.json) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-after.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-after.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-before.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-before.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-table.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-table.png) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.xlsx](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.xlsx) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/sample-scenario-overview.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/sample-scenario-overview.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C19%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202002191739401104-88%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=t22IiWka%2FOW08OYXmDufs5uG9oK5rCJu5GtcYMu3piQ%3D&st=2020-02-19T17%3A35%3A57Z&se=2020-03-21T17%3A40%3A57Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

kbrandl commented on docs/resources/excel-samples.md:
```diff
@@ -1,7 +1,7 @@
 title: 'Sample scripts for Office Scripts in Excel on the web'
 description: 'A collection of code samples to use with Office Scripts in Excel on the web.'
-ms.date: 01/21/2020
+ms.date: 02/13/2020
```
Update this date again?

kbrandl commented on docs/resources/excel-samples.md:
```diff
@@ -132,6 +132,10 @@ async function main(context: Excel.RequestContext) {
 
+## Scenario samples
+
+For samples showcasing larger, real-world solutions, visit [Sample Scenarios for Office Scripts](scenarios/sample-scenario-overview.md).
```
Update casing in article title:

Sample Scenarios for Office Scripts -> Sample scenarios for Office Scripts

kbrandl commented on docs/resources/scenarios/sample-scenario-overview.md:
```diff
@@ -0,0 +1,20 @@
+---
+title: 'Sample Scenarios for Office Scripts'
```
Update casing in article title:  Sample Scenarios for Office Scripts -> Sample scenarios for Office Scripts

Also -- would it make sense to make the titling of this article consistent with that of the [Sample scripts](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/excel-samples?branch=AlexJ-Samples) article -- i.e., `Sample scenarios for Office Scripts in Excel on the web (preview)`

kbrandl commented on docs/resources/scenarios/sample-scenario-overview.md:
```diff
@@ -0,0 +1,20 @@
+---
+title: 'Sample Scenarios for Office Scripts'
+description: 'An explanation for the larger, scenario-driven samples for Office Scripts in Excel on the web.'
+ms.date: 02/13/2020
```
Update the date?

kbrandl commented on docs/resources/scenarios/sample-scenario-overview.md:
```diff
@@ -0,0 +1,20 @@
+localization_priority: Normal
+---
+
+# Sample Scenarios for Office Scripts
```
(same comment as above re updating the title)

kbrandl commented on docs/resources/scenarios/sample-scenario-overview.md:
```diff
@@ -0,0 +1,20 @@
+
+# Sample Scenarios for Office Scripts
+
+Office Scripts help automate parts of your daily routine. These day-to-day tasks often exist in unique ecosystems, with Excel workbooks that are set up in particular ways. These larger scenario samples demonstrate such real-world use-cases. They include both the Office Scripts and the workbooks, so you can see the scenario from end to end.
```
Office Scripts help -> Office Scripts can

kbrandl commented on docs/resources/scenarios/sample-scenario-overview.md:
```diff
@@ -0,0 +1,20 @@
+
+### [Analyze web downloads](analyze-web-downloads.md)
+
+This scenario features a script that parses web traffic records to determine a user's country of origin. It showcases the skills of text parsing, using subfunctions in scripts, applying conditional formatting, and working with tables.
```
Since it's uncommon in the docs for section headings to be hyperlinks, users may not realize that the section header is clickable. Perhaps consider removing the hyperlink from the H3, and instead beginning the sentence with:

`The [Analyze web downloads](analyze-web-downloads.md) scenario features a script that..."`

kbrandl commented on docs/resources/scenarios/analyze-web-downloads.md:
```diff
@@ -0,0 +1,195 @@
+---
+title: 'Office Scripts sample scenario: Internet Traffic Analysis'
```
Suggest making the title here in the metadata consistent with the H1 wording. i.e., "Internet Traffic Analysis" -> "Analyze web downloads"

kbrandl commented on docs/resources/scenarios/analyze-web-downloads.md:
```diff
@@ -0,0 +1,195 @@
+
+# Office Scripts sample scenario: Analyze web downloads
+
+In this scenario, you are tasked with analyzing download reports from your company's website. The goal of this analysis is to determine if the web traffic is coming from the United States or elsewhere in the world.
```
you are -> you're

kbrandl replied:
should you change "United States" -> "United States (US)" here -- so that readers are clear about the meaning of the abbreviation elsewhere in the article?

kbrandl commented on docs/resources/scenarios/analyze-web-downloads.md:
```diff
@@ -0,0 +1,195 @@
+
+In this scenario, you are tasked with analyzing download reports from your company's website. The goal of this analysis is to determine if the web traffic is coming from the United States or elsewhere in the world.
+
+Your colleagues upload the raw data to your workbook. Each week's set of data has its own worksheet. There is also the **Summary** worksheet with a table and chart that shows week-over-week trends.
```
Consider rewording:

Your colleagues upload the raw data to your workbook. Each week's set of data has its own worksheet.

->Your colleagues upload the raw data to your workbook, where each worksheet contains one week of data.

kbrandl commented on docs/resources/scenarios/analyze-web-downloads.md:
```diff
@@ -0,0 +1,195 @@
+
+Your colleagues upload the raw data to your workbook. Each week's set of data has its own worksheet. There is also the **Summary** worksheet with a table and chart that shows week-over-week trends.
+
+The script you've developed in this scenario helps analyze one of these weekly worksheets. It parses each IP address associated with a download and determines whether or not it came from the US. The answer is inserted in the worksheet as a boolean value ("TRUE" or "FALSE") and conditional formatting is applied to those cells. The IP address location results are totaled on the worksheet and copied ...
```
```suggestion
In this scenario, you'll develop a script that analyzes weekly downloads data in the active worksheet. It'll parse the IP address associated with each download and determine whether or not it came from the US. The answer will be inserted in the worksheet as a boolean value ("TRUE" or "FALSE") and conditional formatting will be applied to those cells. The IP address location results will be totaled on the worksheet and copied to the summary table.
```

kbrandl commented on docs/resources/scenarios/analyze-web-downloads.md:
```diff
@@ -0,0 +1,195 @@
+
+## Running the script
+
+Run the **Analyze Web Downloads** script on one of the **Week\*\*** worksheets. This will apply the conditional formatting and location labelling on the current sheet. It also updates the **Summary** worksheet.
```
Some folks may not realize that the way to "run a script on a worksheet" is to: 1) navigate to that sheet, then 2) run the script...maybe reword for clarity? 
-> `Navigate to any of the **Week\*\*** worksheets and run the **Analyze Web Downloads** script.`

kbrandl replied:
This will apply... -> The script will apply...

kbrandl replied:
It also updates... -> It'll also update

kbrandl commented on docs/resources/scenarios/analyze-web-downloads.md:
```diff
@@ -0,0 +1,195 @@
+### Before running the script
+
+![A worksheet that shows raw web traffic data.](../../images/scenario-analyze-web-downloads-before.png)
+
```
For parity between before and after screenshots / results -- do you want to show a screenshot of the summary sheet here in the "Before" section also?

kbrandl commented on docs/resources/scenarios/analyze-web-downloads.md:
```diff
@@ -0,0 +1,195 @@
+
+### After running the script
+
+![A worksheet that shows formatted IP location information with the previous web traffic rows.](../../images/scenario-analyze-web-downloads-after.png)
```
FWIW, the "After" data looks a bit odd to me...I'd expect all the original data to be preserved and new data to be purely additive (not overwriting any of the original data). It feel weird that:
- the original value of the column H header in row 1 has been overwritten (Referrer -> Year),
- the value of column H in the first row of data (row 2) has been overwritten by the new data (http://www.... -> 2020), while original values in column H starting with the second row of data (row 3) are preserved

o365devx commented:
Docs Build status updates of commit _[816b996](https://github.com/OfficeDev/office-scripts-docs/commits/816b996342323da98a0bcd6c585415b255308569)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docfx.json](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/docfx.json) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-after.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-after.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-before.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-before.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-table.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-table.png) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.xlsx](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.xlsx) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/sample-scenario-overview.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/sample-scenario-overview.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C19%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202002192034068470-88%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=zMdw8h9HTN%2BiY0xB%2FEy%2ByJbg%2Fyub5CCpBLczCL0%2FVAs%3D&st=2020-02-19T20%3A30%3A17Z&se=2020-03-21T20%3A35%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

AlexJerabek commented:
Thanks @kbrandl! Updated in response to your feedback. Good catch on the script overwriting data. I've corrected the script and the "after" screenshot.

AlexJerabek commented:
Received some offline feedback from @sumurthy. I'll address it and update the PR soon.

o365devx commented:
Docs Build status updates of commit _[e45dcad](https://github.com/OfficeDev/office-scripts-docs/commits/e45dcadc4c7d3a5187400b8de05e15df1947e41e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docfx.json](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/docfx.json) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-after.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-after.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-before.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-before.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-table.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-table.png) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.xlsx](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.xlsx) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/sample-scenario-overview.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/sample-scenario-overview.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C20%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202002201845083727-88%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=BzK0Qv04gDb9F5yVDOvkB3%2FaQi52e7eBpLtNAQS2Zqc%3D&st=2020-02-20T18%3A41%3A25Z&se=2020-03-22T18%3A46%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

o365devx commented:
Docs Build status updates of commit _[1d7cd2a](https://github.com/OfficeDev/office-scripts-docs/commits/1d7cd2a93f3f9fcfabb95d49f5d55e0778b8c0a3)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/resources/scenarios/grade-calculator.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/grade-calculator.md) | :warning:Warning | | [Details](#user-content-460ece05-02a36218)
[docs/docfx.json](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/docfx.json) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-after.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-after.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-before.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-before.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-table.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-table.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-grade-calculator-after.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-grade-calculator-after.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-grade-calculator-before.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-grade-calculator-before.png) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.xlsx](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.xlsx) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/grade-calculator.xlsx](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/grade-calculator.xlsx) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/sample-scenario-overview.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/sample-scenario-overview.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/toc.yml) | :white_check_mark:Succeeded | |

<a id="460ece05-02a36218"></a>
### [docs/resources/scenarios/grade-calculator.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/grade-calculator.md)
 - **Line 122**: **[Warning]** `````Invalid file link:(~/images/scenario-grade=calculator-after.png).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C20%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202002202013582795-88%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=s3KkEy7af%2FXg3KZENovsrB9km920o%2Blh2hD3kVG2JL0%3D&st=2020-02-20T20%3A10%3A38Z&se=2020-03-22T20%3A15%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

o365devx commented:
Docs Build status updates of commit _[e0376cf](https://github.com/OfficeDev/office-scripts-docs/commits/e0376cf9ea0014c8942df3e60fdb55121e76a46a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docfx.json](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/docfx.json) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-after.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-after.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-before.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-before.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-table.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-table.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-grade-calculator-after.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-grade-calculator-after.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-grade-calculator-before.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-grade-calculator-before.png) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.xlsx](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.xlsx) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/grade-calculator.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/grade-calculator.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/grade-calculator.xlsx](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/grade-calculator.xlsx) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/sample-scenario-overview.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/sample-scenario-overview.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C20%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202002202016255450-88%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Jf2SomXZ%2B112wq7obAGtjEVKiN5ipTSFtQDkZEw%2B%2FlE%3D&st=2020-02-20T20%3A12%3A36Z&se=2020-03-22T20%3A17%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

o365devx commented:
Docs Build status updates of commit _[71d772f](https://github.com/OfficeDev/office-scripts-docs/commits/71d772f304a774514e6fb0f9ee3c692121e2eef2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docfx.json](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/docfx.json) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-after.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-after.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-before.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-before.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-table.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-table.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-grade-calculator-after.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-grade-calculator-after.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-grade-calculator-before.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-grade-calculator-before.png) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.xlsx](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.xlsx) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/grade-calculator.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/grade-calculator.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/grade-calculator.xlsx](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/grade-calculator.xlsx) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/sample-scenario-overview.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/sample-scenario-overview.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C20%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202002202018060692-88%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=12tL6kwo%2B1zvhSrp5SMXGSYbqR%2BerWZzqf8cumPxhWA%3D&st=2020-02-20T20%3A14%3A19Z&se=2020-03-22T20%3A19%3A19Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

o365devx commented:
Docs Build status updates of commit _[0391d99](https://github.com/OfficeDev/office-scripts-docs/commits/0391d99b1561253e024dd5319b46ae673df50a6c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docfx.json](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/docfx.json) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-after.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-after.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-before.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-before.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-analyze-web-downloads-table.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-analyze-web-downloads-table.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-grade-calculator-after.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-grade-calculator-after.png) | :white_check_mark:Succeeded | |
[docs/images/scenario-grade-calculator-before.png](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/images/scenario-grade-calculator-before.png) | :white_check_mark:Succeeded | |
[docs/resources/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/excel-samples.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.xlsx](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/analyze-web-downloads.xlsx) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/grade-calculator.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/grade-calculator.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/grade-calculator.xlsx](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/grade-calculator.xlsx) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/sample-scenario-overview.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/resources/scenarios/sample-scenario-overview.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Samples/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C20%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202002202022035006-88%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=iTHhz9kHIQdHAImD%2Fa3RJnubW1KQAxVQCT%2FEkk%2BucTw%3D&st=2020-02-20T20%3A18%3A17Z&se=2020-03-22T20%3A23%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

AlexJerabek commented:
@jedunn signed off in person. Merging!
